src/lager.erl

```diff
@@ -594,3 +595,34 @@ pr_stacktrace(Stacktrace) ->
 pr_stacktrace(Stacktrace, {Class, Reason}) ->
     lists:flatten(
         pr_stacktrace(Stacktrace) ++ "\n" ++ io_lib:format("~s:~p", [Class, Reason])).
+
+rotate_file(FileName) ->
+    Handlers = lager_config:global_get(handlers),
+    RotateHandlers = lists:filter(
+        fun ({{lager_file_backend, FN}, _, _}) ->
```

Comment:
There is no standard file-wrapping handler spec. So It's unsafe to assume any handler with 2-element tuple to be a file handler.
By exposing `rotate_handler`, do you mean something like this:
```
rotate_handler(Backend) ->
    rotate_handler(Sink, Backend).
rotate_handler(Sink, Backend) ->
    gen_event:call(Sink, Backend, rotate, ?TIMEOUT).
```